dmutils/cookie_probe.py

```diff
@@ -0,0 +1,28 @@
+from datetime import timedelta
+
+
+DEFAULT_DM_COOKIE_PROBE_COOKIE_NAME = "DM_COOKIE_PROBE"
+DEFAULT_DM_COOKIE_PROBE_COOKIE_VALUE = "yum ouch"
```

Comment:
Not sure about this...

Reply:
"cookie" -> "yum"
"probe" -> "ouch"

Might just make it "yum"